### PR TITLE
Review Accounts tooltips and fix error with Storage/Pools "exec" tooltip

### PR DESCRIPTION
**Pull request by Mrt134**
Reviewed 20/20 Accounts forms tooltips and adjusted for brevity.
Adjusted some field names slightly for accuracy/simplicity.
Fixed reported error with "Exec" tooltip in Storage/Pools form.
Local build testing: no issues.

### CODE REVIEW

File: src/app/pages/account/users/user-form/user-form.component.ts [hunk 1]
```diff
@@ -43,10 +43,12 @@ export class UserFormComponent {
           type : 'input',
           name : 'username',
           placeholder : T('Username'),
-          tooltip : T('Enter an alphanumeric username of 8 characters or less.\
-            Usernames can not begin with a hyphen (<b>-</b>) or contain a\
-            space, tab, or these characters: <b>, : + & # %^ ( ) ! @ ~ * ? < > =</b> .\
-            A <b>$</b> can only be used as the last character.'),
+          tooltip : T('Enter an alphanumeric username of 8 characters '
+                    + 'or less. Usernames can not begin with a hyphen '
```
Comment (wblock) [inline src/app/pages/account/users/user-form/user-form.component.ts]: s/can not/cannot/

File: src/app/pages/account/users/user-form/user-form.component.ts [hunk 2]
```diff
@@ -202,24 +207,24 @@ export class UserFormComponent {
...
+                    + 'target="_blank">sudo</a>.'),
           isHidden: false
         },
         {
           type : 'checkbox',
           name : 'microsoft_account',
           placeholder : T('Microsoft Account'),
-          tooltip : T('Check if the user will be connecting from a\
-            Windows 8 or higher system.'),
+          tooltip : T('Check when the user connects from a '
```
Comment (wblock) [inline src/app/pages/account/users/user-form/user-form.component.ts]: After consultation with John:
'Set to allow additional username authentication methods when the user is connecting from a Windows 8 or newer operating system.'

**Comment by Mrt134** [comment]
Sure, we can look into it. Is there a different way to test the extraction utility functionality? When Warren and I were working on updating the syntax, we didn't see any problems with just the npm install/start testing.

**Comment by erinix** [comment]
These look fine to me, what says you @wblock ?

**Pull request merged**